src/EnergyEfficientAgent.cpp

```diff
@@ -249,10 +248,6 @@ namespace geopm
                                         "(): region exit before entry detected.",
                                         GEOPM_ERROR_RUNTIME, __FILE__, __LINE__);
                     }
-                    if (last_region_info.runtime != 0.0 &&
-                        last_region_info.runtime < M_MIN_LEARNING_RUNTIME) {
-                        last_region_it->second->disable();
```

Comment:
looks like this is the only use of the EnergyEfficientRegion::disable() api.  This method should be deleted from the class.  What was the check for `last_region_info.runtime != 0.0` for?  Why is deleting it okay?

Reply:
0.0 is the starting runtime for regions prior to first observation, the check was there to prevent disabling a region before it has ran.

yes this is the only place disable is used, there is a todo in the commit message.  I was unsure of how big of a change we wanted here an now.  if the removal of the api is a blocker for merge then I can take care of that now.